utils/kata-manager.sh

```diff
@@ -687,6 +687,9 @@ configure_containerd()
 	sudo test -e "$containerd_config" || {
 		sudo touch "$containerd_config"
 		info "Created $containerd_config"
+		cat <<-EOF | sudo tee -a "$containerd_config"
+		version = 2
+		EOF
```

Comment:
kata-manager should actually check which version of config is expected by containerd (`containerd config default` gives you such info) and then do the changes in the proper way.

https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/scripts/kata-deploy.sh#L596 for some inspiration.